design/FY2017/NFP-UpdateLEEDSummaryReport.md

```diff
@@ -0,0 +1,1386 @@
+ - ElectricEquipment
+ - GasEquipment
+ - HotWaterEquipment
+ - SteamEquipment
+ - OtherEquipment
+ - ElectricEquipment:ITE:AirCooled
+ - ZoneBaseboard:OutdoorTemperatureControlled
+ - Exterior:Lights
+ - Exterior:FuelEquipment
+ - Exterior:WaterEquipment
+ - Fan:SystemModel
+ - Fan:ConstantVolume
+ - Fan:VariableVolume
+ - Fan:OnOff
+ - Fan:ZoneExhaust
+ - Fan:ComponentModel
+ - WaterHeater:Stratified
+ - EnergyManagementSystem:MeteredOutputVariable
+ - Refrigeration:CompressorRack
+ - Refrigeration:Condenser:AirCooled
+ - Refrigeration:Condenser:EvaporativeCooled
+ - Refrigeration:Condenser:WaterCooled
+ - Refrigeration:GasCooler:AirCooled
+ - Refrigeration:Compressor
+ - Refrigeration:System
+ - Refrigeration:TranscriticalSystem
+ - Refrigeration:SecondarySystem
+ - WaterUse:Equipment
+
+And add End-Use Subcategory fields for other selected objects that should have them:
+
+ - Boiler:HotWater
+ - Boiler:Steam
+ - Chiller:Electric:EIR
+ - Chiller:Electric:ReformulatedEIR
+ - Chiller:Electric
+ - CoolingTower:SingleSpeed
+ - CoolingTower:TwoSpeed
+ - WaterHeater:Mixed
+
+Change the following or similar field:
+
+```
+  A5 , \field End-Use Subcategory
+       \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
+       \note The following special tags will also place the end-use in specific rows in the LEED Summary table
+       \note EAp2-4/5. Performance Rating Method Compliance:
+       \note Fans-Parking Garage, Interior Lighting-Process, Cooking, Industrial Process, Elevators and Escalators.
+       \type alpha
+       \retaincase
+       \default General
+```
+
+To:
+
+```
+  A5 , \field End-Use Subcategory
+       \note Used to categorize the end-uses in the ABUPS End Uses by Subcategory table.
+       \note Some of the end-use uses (prior to General) are used specifically in the LEED summary table.
+       \note When using an end use not on this list select Other and define the end use in the next field.
+       \type choice
+       \key FansParkingGarage
+       \key FansKitchenVentilation
+       \key InteriorLightingProcess
+       \key LightingInApartments
+       \key Cooking 
+       \key IndustrialProcess
+       \key ElevatorsEscalators
+       \key ITequipment
+       \key RefrigerationEquipmentUnregulated
+       \key RefrigerationEquipmentRegulated
+       \key BuildingTransformers
+       \key General
+       \key ITECPU
+       \key ITEFans
+       \key ITEUPS
+       \key Computers
+       \key OfficeEquipment
+       \key GroundsLights
+       \key FacadeLighting
+       \key ParkingLighting
+       \key AccentLighting
+       \key TaskLights
+       \key ClothesWashing
+       \key ClothesDrying
+       \key ZoneExhaustFans
+       \key FumeHoods
+       \key Bathrooms
+       \key Dishwashing
+       \key Showers
+       \key BathroomHandwashing
+       \key KitchenSink
+       \key LightFoodPrep
+       \key RainAndWellWater
+       \key Other
+       \default General
```

Comment:
Why would you include all these \key choices for a chiller, or boiler, or...
Since this is a change in the IDD then don't you have flexibility to include only those that are applicable? I realize we like to be general, but if I were a user I'd ask myself why this list is so long for a chiller that would have nothing to do with lighting, or fans, or transformers, etc.

    Refrigeration:SecondarySystem,
       A9;  \field End-Use Subcategory
         \type alpha
         \retaincase
         \default General

    Chiller:Electric:EIR,
       A16;  \field End-Use Subcategory
         \type alpha
         \retaincase
         \default General